src/tools/pub/pubspec.md

```diff
@@ -184,8 +184,7 @@ authors:
 - Bob Nystrom <rnystrom@google.com>
 {% endprettify %}
 
-If anyone uploads your package to the Pub site, your email address is
-public.
+If anyone uploads your package to the Pub site, your email address is public. The email can be from any email provider as long as it being listed publicly is okay.
```

Comment:
The “listed publicly is okay” text seems unenforceable. How about: 

The email addresses can be from any provider.
If anyone … site,
then the author information (including email addresses) becomes public.

Reply:
Will change

Reply:
@Stargator once you change this, I’ll happily merge this PR.

Reply:
@Stargator I'm planning to change this Friday unless you get to it first.

Reply:
@kwalrath Feel free. I'm going to be busy tonight and during the day on Friday.